master/internal/config/config.go

```diff
@@ -236,67 +237,42 @@ type ObservabilityConfig struct {
 	EnablePrometheus bool `json:"enable_prometheus"`
 }
 
-func readPreemptionFromScheduler(conf *resourcemanagers.SchedulerConfig) *bool {
-	if conf == nil || conf.Priority == nil {
-		return nil
-	}
-	return &conf.Priority.Preemption
-}
-
 func readPriorityFromScheduler(conf *resourcemanagers.SchedulerConfig) *int {
 	if conf == nil || conf.Priority == nil {
 		return nil
 	}
 	return conf.Priority.DefaultPriority
 }
 
-// ReadPreemptionStatus resolves the desired preemption status for a job.
-func ReadPreemptionStatus(rpName string, jobConf interface{}) bool {
+// ReadRMPreemptionStatus resolves the preemption status for a resource manager.
+func ReadRMPreemptionStatus(rpName string) bool {
```

Comment:
Should we call it `ReadRPPreemptionStatus` ?

Reply:
hmm that works too. As this is implemented in the case of k8 it doesn't really look at the RP that's why I stayed away from it 🤔